impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -320,10 +321,10 @@ void ContextVK::Setup(Settings settings) {
   }
 
   auto sampler_library =
-      std::shared_ptr<SamplerLibraryVK>(new SamplerLibraryVK(device.get()));
+      std::shared_ptr<SamplerLibraryVK>(new SamplerLibraryVK(device_.get()));
```

Comment:
I didn't look into it but, I suspect this guy and the FenceWaiterVK may need device holders too.